docs/faq.md

```diff
@@ -0,0 +1,82 @@
+
+# Tool Kit FAQs
+
+Below are responses to some general questions about Tool Kit that have been asked during migrations. If your question is not answered here please ask in the [#tool-kit](https://app.slack.com/client/T025C95MN/C02TRE2V2Q1/thread/C042NBBTM-1671107986.307219) slack channel and consider adding it, and the answer, to this document.
+
+## How do I add package specific configuration (for e.g. ESLint) to the Tool Kit plugin (e.g. @dotcom-tool-kit/eslint))?
+
+As much as possible Tool Kit doesn't handle configuration for third party packages. The purpose of Tool Kit’s plugin for packages is to get them working in the FT development workflow with the minimum possible configuration. Tool Kit plugins will leave package configuration to a specific configuration file for that package (e.g. `.eslintrc` for the ESLint package).
+
+## How can I use custom commands for my tooling that are not accommodated for by Tool Kit's config?
+
+You don't need to use Tool Kit for everything. Tool Kit handles common tooling use cases that are required for most apps to work in the Customer Products ecosystem, but it's fine for your app to include additional tooling features that don't use Tool Kit at all.
+
+See [How do I run watch mode for tests with Tool Kit?](#how-do-i-run-watch-mode-for-tests-with-tool-kit) for a specific example of calling a third party package that Tool Kit configures without using Tool Kit at all.
+
+## How do I run watch mode for tests with Tool Kit?
+
+Now that we manage tasks with npm scripts, you can define a script to run your tests in watch mode:
+
+```json
+{ 
+    "scripts": 
+        {
+            "watch-tests": "jest --watch" 
+        } 
+}
+```
+
+Then run `npm watch-tests` to execute your script.
+
+You can also use the package directly with npx to run your test runner with any options you wish:
+
+```sh
+npx jest path/to/tests --watch
+```
+
+Note, this is an example of a script that doesn't use Tool Kit at all, and that's fine.
+
+## Is there a central list of all hooks and tasks available in Tool Kit?
+
+Funny you should ask! We have just the thing:
+
+| Hook                | Hook exported by plugin | Possible tasks       |
```

Comment:
**praise:** This is a beautiful table 🤩 Maybe we should run prettier / [remark](https://github.com/remarkjs/remark) to format our documentation.